roles/openshift_etcd_migrate/templates/validatemigration.py.j2

```diff
@@ -0,0 +1,91 @@
+from subprocess import PIPE, Popen
```

Comment:
I think we can get rid of the templating, can we not just use /etc/etcd/peer.crt /etc/etcd/peer.key and /etc/etcd/ca.crt I think those will be the same regardless of configuration? and then just use `hostname` for the etcdctl commands